src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DNodeViewModel.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Windows;
+using Autodesk.DesignScript.Interfaces;
+using Dynamo.Models;
+using Dynamo.Utilities;
+
+namespace Dynamo.Wpf.ViewModels.Watch3D
+{
+    public class HelixWatch3DNodeViewModel : HelixWatch3DViewModel
+    {
+        private NodeModel node;
+
+        public static HelixWatch3DNodeViewModel Start(NodeModel node, Watch3DViewModelStartupParams parameters)
```

Comment:
Consider moving Watch3DViewModelStartupParams in as an internal class of `Watch3DViewModel`.